src/BaGetter/Startup.cs

```diff
@@ -30,8 +31,17 @@ public void ConfigureServices(IServiceCollection services)
         services.ConfigureOptions<ValidateBaGetterOptions>();
         services.ConfigureOptions<ConfigureBaGetterServer>();
 
-        services.AddAuthentication(AuthenticationConstants.NugetBasicAuthenticationScheme)
-            .AddScheme<AuthenticationSchemeOptions, NugetBasicAuthenticationHandler>(AuthenticationConstants.NugetBasicAuthenticationScheme, null);
+        services.AddAuthentication(options =>
+        {
+            // Breaks existing tests if the contains check is not here.
```

Comment:
If possible, find out why the test tries to initialize this multiple times and fix it there, I'd like to avoid test code in the application code.

Reply:
```
WebApplicationFactory behavior:
When WebApplicationFactory.CreateClient() is called, it creates a new test server, which involves rebuilding the application. This process includes calling ConfigureServices again, which would lead to AddAuthentication being called a second time.
```

Reply:
As far I found out, this is propably the cleanest solution around that.

Reply:
Thanks for checking.